.github/workflows/release.yml

```diff
@@ -1,15 +1,28 @@
 on:
+  # You must provide a tag either via the action input or the git ref (i.e push / create a tag).
+  # If you do not provide a tag the action will fail.
+
+  # If the tag of the release you are creating does not yet exist, you should set
+  # both the tag and commit action inputs. commit can point to a commit hash or a branch name (ex - main).
+#   workflow_dispatch:
+#     inputs:
+#       branch:
+#         description: 'Define branch name to run the release GH action against'
+#         required: true
+#         default: 'main'
   release:
-    types: [published]
+    type: [created]
+  push:
+    tags:
+      - v1.*
```

Comment:
Does it bring any benefit triggering the release in both `push.tags.v1.*` and `release.type.created`? What if we only leave one?

Reply:
It will trigger if one condition is met and won't run trigger a second run if both are met (as far as I understand from the docs). This would allow a maintainer to locally create and push a tag as was being done in the past and still have the new automation complete.

Reply:
> Another comment I forgot: I guess we don't need the `package version` anymore. Could you remove it?

This has been addressed with removing the `pkg/version` directory and switching to declaring it in flannel.go